manifests/o/o2sh/onefetch/2.14.1/o2sh.onefetch.installer.yaml

```diff
@@ -0,0 +1,20 @@
+# Created with YamlCreate.ps1 v2.2.1 using InputObject 🤖 $debug=QUSU.CRLF.7-2-7
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: o2sh.onefetch
+PackageVersion: 2.14.1
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+UpgradeBehavior: install
+ReleaseDate: 2022-11-27
+AppsAndFeaturesEntries:
+- DisplayName: onefetch version v2.13.2
+  DisplayVersion: v2.13.2
```

Comment:
These are the values that need updating

Reply:
Yes, I removed them as they're not needed anymore.

Reply:
The ProductCode would still be worth keeping as it's the same through releases, so there wouldn't be any problem with automation. I can always add this at a later date though